documents/src/pages/elements/datetime-picker.md

```diff
@@ -343,6 +344,288 @@ section {
 </ef-datetime-picker>
 ```
 
+## Custom cells
+Datetime-picker allows you to customise cells on the calendar using slots. Slot name could begin with either `from-` or `to-` as a prefix, indicating which side of the calendar to be customised. After the prefix, you need to specify which date, month, or year to be customised with `yyyy-MM-dd`, `yyyy-MM`, or `yyyy`. If the prefix is not defined, the slot will apply to from calendar.
+
+::
+```javascript
+::datetime-picker::
+const datetimePicker = document.querySelector('ef-datetime-picker');
+const holidays = ['2023-04-07', '2023-04-10', '2023-05-01', '2023-05-18', '2023-05-29'];
+
+datetimePicker.filter = date => !holidays.includes(date);
+datetimePicker.views = ['2023-04','2023-05'];
+```
+```html
+<section>
+  <div>
+    <h5>Germany Public Holidays 2023</h5>
+    <h6>April</h6>
+    <p>
+      07: Good Friday<br>
+      10: Easter Monday
+    </p>
+    <h6>May</h6>
+    <p>
+      01: Labour Day<br>
+      18: Ascension Day<br>
+      29: White Monday
+    </p>
+  </div>
+  <ef-datetime-picker id="duplex-datetime-picker" opened duplex range lang="de">
+    <div class="holiday" slot="from-2023-04-07">7</div>
+    <div class="holiday" slot="from-2023-04-10">10</div>
+    <div class="holiday" slot="from-2023-05-01">1</div>
+    <div class="holiday" slot="from-2023-05-18">18</div>
+    <div class="holiday" slot="from-2023-05-29">29</div>
+    <div class="holiday" slot="to-2023-04-07">7</div>
+    <div class="holiday" slot="to-2023-04-10">10</div>
+    <div class="holiday" slot="to-2023-05-01">1</div>
+    <div class="holiday" slot="to-2023-05-18">18</div>
+    <div class="holiday" slot="to-2023-05-29">29</div>
+  </ef-datetime-picker>
+</section>
+```
+
+```css
+ef-datetime-picker {
+  --holiday-background-color: #F5475B;
+  --holiday-cell-color: #fff;
+}
+
+div, ef-datetime-picker {
+  margin: 20px
+}
+
+section {
+  display: flex;
+  height: 290px;
+  padding: 0 3px;
+}
+
+h5, h6 {
+  margin-top: 0px;
+}
+
+ef-datetime-picker .holiday {
+  background-color: var(--holiday-background-color);
+  color: var(--holiday-cell-color);
+  width: 100%;
+  height: 100%;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+```
+::
+
+For more advanced use cases, you can use `before-cell-render` event to get [CalendarCell](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/calendar/types.ts) to style your slotted cell. The event is fired before and every time each cell is rendered. From Calendar and to Calendar could be distinguished by their id: `calendar` and `calendar-to` respectively.
```

Comment:
Just noticed that we don't capitalize our element name. Let's change to this one.
```suggestion
For more advanced use cases, you can use `before-cell-render` event to get [CalendarCell](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/calendar/types.ts) to style your slotted cell. The event is fired before and every time each cell is rendered.  "From" calendar and "to" calendar could be distinguished by their id: `calendar` and `calendar-to` respectively.
```

Reply:
Should we change "From" to lowercase?

Reply:
Yeah, it's pretty awkward. Let's use this one instead.
```suggestion
For more advanced use cases, you can use `before-cell-render` event to get [CalendarCell](https://github.com/Refinitiv/refinitiv-ui/blob/v7/packages/elements/src/calendar/types.ts) to style your slotted cell. The event is fired before and every time each cell is rendered. To distinguish between "from" calendar and "to" calendar, check for their id: `calendar` and `calendar-to` respectively.
```